### ISSUE TITLE
understanding scoreDetails.authorMatchScore maximum value

### ISSUE THREAD

**Issue by abubelinha** [gnames/gnverifier#86]
http://verifier.globalnames.org/api/v0/verifications/Isoetes%20cf.%20longissima%20Bory?data_sources=196|197|198|158&all_matches=true

Shouldn't the value of authorMatchScore equal 1 when provided author's name is identical to datasource author's name?

I couldn't find any examples where the returned score is higher than 0.71428573

**Comment by dimus** [comment]
For botanical names it is true, the author score never goes to one. A name needs a matching year for both names to get to 1.
It is needed because for zoological names authors matching year perfectly should be higher than all other cases.

**Comment by abubelinha** [comment]
Ah!!  Understood

So for my own needs we can say the botanical scoring ranges between 0 and 0.71428573? (so I can calibrate my results)

It's a strange maximum value. I suppose it's the result of some fraction, sqroot or whatever?

**Comment by dimus** [comment]
The score was made at first for internal use only to determine the best result. The way it works is by taking a very big 32bit number xxxxxxxx-xxxxxxxx-xxxxxxxx-xxxxxxxx and assign each score part a place of 1-3 bits on this number.
For example currently accepted names takes one bit 0 or 1, because it has only 2 states, and authorship has 8 states, so it takes 3 bits 000, 001, 010... 111

Every score than placed on the 32 bit binary number according to its priority, things on the left tump things on the right. The resulting bit 32bit number is used for sorting. So the final result looks somethng like 

00011000-1000000-01101000-00000000

The algorithm was for intenal use, but then I found that people do not understand that there is author matching happening as well as other things, so I tried to make sense of this score in the output to show details of matching.

I need now to figure out how to assign weight to each scoring feature, so I can produce a more understandable total number from 0.0 to 1.0 as a total score.
